rest-api-spec/src/main/resources/rest-api-spec/api/_internal.get_desired_balance.json

```diff
@@ -0,0 +1,23 @@
+{
+  "_internal.get_desired_balance":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/get-desired-balance.html",
+      "description": "Gets the latest desired balance and basic metrics. Designed for indirect use by ECE/ESS and ECK. Direct use is not supported."
```

Comment:
I believe this is common for an internal api to mention ECE/ESS. Though I am not sure we should do it here as well.

Reply:
I used this just a stock disclaimer for internal APIs. Would some different wording better here?

Reply:
We can keep it experimental.

Also, we can instead say something like

"This API is a diagnostics API and the output should not be relied upon for building applications."

No need to mention the cloud/ece/eck, desired balance is for all.